text/0000-chmod.md

```diff
@@ -0,0 +1,62 @@
+# Chmod Buildpack
+
+## Summary
+
+A chmod buildpack that allows users to change directories and files (produced at buildtime) UNIX permissions.
+
+This buildpack would be configured to run last, so that it would be able to change any directories and files produced by previous buildpacks.
+
+## Motivation
+
+Buildstacks and Builders rely on Stacks.
+
+Stacks define build time and runtime base images.
+
+Some stacks do not use the same user / UID between build and run images, [for security reasons](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0045-user-ids.md), since a runtime user should just be able to run a process; not create directories nor install packages.
+
+Unfortunately, some buildpacks do need to create directories and files: they run setup scripts during runtime that fail because they can't write folders nor files, since they were run with a runtime user, who can be restricted with some Stacks. 
```

Comment:
```suggestion
Unfortunately, some applications, like `dist-zip` archives, do need to create directories and files: they run setup scripts during runtime that fail because they can't write folders nor files, since they were run with a runtime user, who can be restricted with some Stacks. 
```

This is a small technicality, but I'd say "application" here because buildpacks only run at build time.